doc/mob_skill_db.md

```diff
@@ -0,0 +1,223 @@
+
+## Field explanation
+
+### Monster_Constant
+The monster's name constant, found in [mob_db.conf](../db/re/mob_db.conf) as *SpriteName*.  
+There are 3 special constants for global skill assignment:  
+* **ALL_MOBS**: Add skills to all monsters.
+* **ALL_MOBS_BOSS**: Add skills to all boss monsters.
+* **ALL_MOBS_NONBOSS**: Add skills to all non-boss monsters.
+
+### Skill_Constant
+The skill's name constant, found in [skill_db.conf](../db/re/skill_db.conf) as *Name*.  
+Note: You can add multiple Skill_Constant blocks.
+
+### ClearSkills
+If set to `true`, all previously defined skills for this monster will be removed.
+
+### SkillLevel
+The skill level which should be used.  
+Minimum value is `1`. Maximum value is `mob_max_skilllvl` from [conf/map/battle/skill.conf](../conf/map/battle/skill.conf).
+
+### SkillState
+Defines in which state the monster is able to cast the skill.
+State         | Description
+:------------ | :----------
+MSS_ANY       | Monster is in any state except MSS_DEAD.
+MSS_IDLE      | Monster has no target and isn't walking.
+MSS_WALK      | Monster is walking.
+MSS_LOOT      | Monster is looting or walking to loot.
+MSS_DEAD      | Monster is dying.
+MSS_BERSERK   | Monster is attacking after starting the battle.
+MSS_ANGRY     | Monster is attacking after being attacked.
+MSS_RUSH      | Monster is following an enemy after being attacked.
+MSS_FOLLOW    | Monster is following an enemy without being attacked.
+MSS_ANYTARGET | Same as MSS_ANY but monster must have a target.
+
+### SkillTarget
+Defines the skill's target.
+Target      | Description
+:---------- | :----------
+MST_TARGET  | The monster's current target.
+MST_RANDOM  | A random enemy within skill range.
+MST_SELF    | The monster itself.
+MST_FRIEND  | A random friend within skill range. If no friend was found, MST_SELF is used.
+MST_MASTER  | The monster's master. If no master was found, MST_FRIEND is used.
+MST_AROUND1 | Random cell within a range of 1. (Affects ground skills only.)
+MST_AROUND2 | Random cell within a range of 2. (Affects ground skills only.)
+MST_AROUND3 | Random cell within a range of 3. (Affects ground skills only.)
+MST_AROUND4 | Random cell within a range of 4. (Affects ground skills only.)
+MST_AROUND5 | Same as MST_AROUND1, but the monster's current target must be in skill range.
+MST_AROUND6 | Same as MST_AROUND2, but the monster's current target must be in skill range.
+MST_AROUND7 | Same as MST_AROUND3, but the monster's current target must be in skill range.
+MST_AROUND8 | Same as MST_AROUND4, but the monster's current target must be in skill range.
+MST_AROUND  | Same as MST_AROUND4.
+
+### Rate
+The chance of successfully casting the skill if the condition is fulfilled. (10000 = 100%)  
+Minimum value is `1`. Maximum value is `10000`.
+
+### CastTime
+The skill's cast time in milliseconds.  
+Minimum value is `0`. Maximum value is `MOB_MAX_CASTTIME` from [src/map/mob.c](../src/map/mob.c).
```

Comment:
this `MOB_MAX_CASTTIME` doesn't seem like exists in source.

Reply:
@Emistry It's part of #2691.